### PR TITLE
feat: inline warning badges

### PR DESCRIPTION
**Pull request by nghanyi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,20 +17,25 @@
...
     "@project-serum/anchor": "^0.26.0",
+    "@radix-ui/react-popover": "^1.1.1",
     "@solana/spl-token": "0.1.8",
     "@solana/spl-token-registry": "~0.2.4574",
     "@solana/wallet-adapter-wallets": "0.19.22",
     "@solana/web3.js": "1.87.6",
     "@tanstack/react-query": "^4.36.1",
     "bn.js": "5.2.1",
     "bs58": "5.0.0",
+    "class-variance-authority": "^0.7.0",
```
Comment (worlddlckgh) [inline package.json:30]: dont think these are used, please remove
- class-variance-authority
- lucide-react
- tailwindcss-animate

File: src/components/SuggestionTags/hooks/useSuggestionTags.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+import JSBI from 'jsbi';
+import { checkIsUnknownToken } from 'src/misc/tokenTags';
+import { UnknownTokenSuggestion } from '../Tags/UnknownTokenSuggestion';
+import { LSTSuggestion } from '../Tags/LSTSuggestion';
+import { AuthorityAndDelegatesSuggestion } from '../Tags/AuthorityAndDelegatesSuggestion';
+import { TransferTaxSuggestion } from '../Tags/TransferTaxSuggestion';
+import { PriceWarningSuggestion } from '../Tags/PriceWarningSuggestion';
+import { DCASuggestion } from '../Tags/DCASuggestion';
+import { extractTokenExtensionsInfo } from '../Tags/Token2022Info';
+import { useUSDValue } from 'src/contexts/USDValueProvider';
```
Comment (worlddlckgh) [inline src/components/SuggestionTags/hooks/useSuggestionTags.tsx:19]: Hi, we only need to support a subset of warnings as mentioned in the ticket.
```
AuthorityAndDelegatesSuggestion
PriceWarningSuggestion
TransferTaxSuggestion
UnknownTokenSuggestion
```

Comment (worlddlckgh) [reply src/components/SuggestionTags/hooks/useSuggestionTags.tsx:19]: remember to drop unused files

File: package.json (hunk 1 above)
Comment (nghanyi) [reply package.json]: removed the dependencies

File: src/components/SuggestionTags/hooks/useSuggestionTags.tsx (hunk 2 above)
Comment (nghanyi) [reply src/components/SuggestionTags/hooks/useSuggestionTags.tsx]: removed used files

**Comment by worlddlckgh** [comment]
Thanks for the contrib, taking over here
https://github.com/jup-ag/terminal/pull/107

**Pull request closed without merging**